crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -127,6 +128,10 @@ struct DsnArgs {
     /// Known external addresses
     #[arg(long, alias = "external-address")]
     external_addresses: Vec<Multiaddr>,
+    /// Defines endpoints for the prometheus metrics server. It doesn't start without at least
+    /// one specified endpoint. Format: 127.0.0.1:8080
+    #[arg(long, alias = "metrics-endpoint")]
+    metrics_endpoints: Vec<SocketAddr>,
```

Comment:
Shouldn't be in DSN configuration, these are generic metrics that will be used to expose various parameters of the farmer as well.